docs/src/hal/tools.adoc

```diff
@@ -151,7 +151,7 @@ operating instructions should be located here eventually. For now,
 refer to section  <<sec:tutorial-halscope>> in the tutorial chapter,
 which explains the basics.
 
-The halscope File menu selector provides buttons to save a configuration
+The halscope "File" menu selector provides buttons to save a configuration
```

Comment:
I think this would be more common:
```suggestion
The halscope file menu selector provides buttons to save a configuration
```

Reply:
I saw it but then thought that the author should have that freedom and the original is formally correct.

Reply:
But `"File" menu` looks a bit weird and never seen it like this.

Reply:
I do not want to change it, but won't intervene if you are insisting on changing it. IMHO this only distracts from other problems that sentence has.

Reply:
I would say F/file menu or menu "File"

Reply:
Ah, I did not recall to have introduced those quotes. Hm. Right.

Reply:
Wikipedia has it as 'File menu' (without any quotes), so I went for that.  Thank you!

Reply:
:laughing: :see_no_evil:

Reply:
What about

```suggestion
The `halscope` 's `menu:File[Save configuration...] menu entry allows to save a configuration
```

Reply:
I think that is overdoing it.

Reply:
Having a language allowing us to semantically markup elements specific to software UIs is a valuable strength IMHO.
Menus (and navigation within) as well as buttons need a defined formatting anyway. RH uses AsciiDoc special macros and puts any UI element in bold, whereas GitLab uses the ``*`Menu > Entry...`*`` pattern for navigation.

Halscope is either an application name or a command name. In the latter case it should be inline code blocked. In the former, we have to decide if we put some kind of emphase on applications and products names. Finally we have to devise (a) rule(s) whether to use one or the other depending on context.